src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/TypedReferenceHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Internal.Runtime.CompilerHelpers
+{
+    /// <summary>
+    /// These methods are used to implement TypedReference-related instructions.
+    /// </summary>
+    internal static class TypedReferenceHelpers
+    {
+        public unsafe static RuntimeTypeHandle TypeHandleToRuntimeTypeMaybeNull(RuntimeTypeHandle typeHandle)
```

Comment:
You don't have to worry about it because this is something RyuJIT needs as part of compiling `refanytype` IL instruction. The UTC team showed little to no enthusiasm about implementing that so what we'll have to do on Project N is update DR (or GateKeeper) to mark all methods that have this (and related instructions) as unsupported IL and bail.